bhclub.html

```diff
@@ -52,7 +53,26 @@ <h1>Events</h1>
             <h1>Teams</h1>
         </section>
         <section id="contact">
-            <h1>Contact</h1>
+            <div>
+                <h1>Contact</h1>
+                <p>Have any questions? Feel free to reach out to us at <a href="mailto:hello@brickhack.org">hello@brickhack.org</a></p>
+                <p>You can also sign up to receive newsletters <a href="./index.html">here</a>.</p>
+            </div>
+            <div id="footer">
```

Comment:
I don't think it makes sense for the footer to be a `div` inside the contact `section`. This should be a `footer` element after the `section`.

Reply:
Design wants the footer to be inside of the section, because it's all contact information